manifests/kvm.pp

```diff
@@ -33,14 +42,30 @@
       # Enable Forwarding
       'net.ipv4.conf.all.forwarding': value => '1';
       'net.ipv4.ip_forward':          value => '1';
+    }
+
+    unless $facts['libvirt_br_netfilter_loaded'] {
+      if $load_kernel_modules {
+        kmod::load { 'br_netfilter': }
+
+        Kmod::Load['br_netfilter'] -> Sysctl['net.bridge.bridge-nf-call-arptables']
```

Comment:
Why not regular metaparameters on the `kmod::load`?

Reply:
Consistency with the other chaining for visual scanning.